src/cart/crt_rpc.c

```diff
@@ -1139,7 +1139,8 @@ crt_req_send(crt_rpc_t *req, crt_cb_t complete_cb, void *arg)
 	 * function.  Referenced dropped at end of this function.
 	 */
 	RPC_ADDREF(rpc_priv);
-
+	/* Insert the local job id in the rpc request. */
+ 	rpc_priv->crp_req_hdr.cch_clid = crt_gdata.cg_clid;
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs